packages/python/m/ci/docker/gh_workflow_multi.py

```diff
@@ -72,7 +72,7 @@
   manifest:
     runs-on: {default_runner}
     needs: [blueprints, build]
-    strategy:
+    strategy:{manifest_strategy_options}
```

Comment:
can this be an `enum`? GH lists the valid options for this right?

Reply:
No point on doing this since the workflow is generated. I don't think there's many other options maybe one but not needed for now: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategy

Reply:
got it